ui/main/src/app/modules/feed/components/card-list/card-list.component.ts

```diff
@@ -34,9 +59,65 @@ export class CardListComponent implements AfterViewChecked, OnInit {
 
     adaptFrameHeight() {
         const rect = this.domCardListElement.getBoundingClientRect();
-        const height = window.innerHeight - rect.top - 10;
-        this.domCardListElement.style.height = `${height}px`;
+        const height = window.innerHeight - rect.top - 30;
+        this.domCardListElement.style.maxHeight = `${height}px`;
```

Comment:
if  this.hideAckAllCardsFeature is true , height shall be top -10 instead of 30

Reply:
Done.